app/src/main/java/org/sopt/soptseminar_week1/RepositoryListAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.soptseminar_week1
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.soptseminar_week1.databinding.ItemRepositoryListBinding
+
+class RepositoryListAdapter :
+    RecyclerView.Adapter<RepositoryListAdapter.RepositoryListViewHolder>() {
+    val repositoryList = mutableListOf<RepositoryListInfo>()
+
+    class RepositoryListViewHolder(private val binding: ItemRepositoryListBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun onBind(repositoryListInfo: RepositoryListInfo) {
+            binding.textName.text = repositoryListInfo.repositoryName
+            binding.textDesc.text = repositoryListInfo.repositoryDesc
+            binding.textLang.text = repositoryListInfo.repositoryLang
```

Comment:
```suggestion
           with(binding){
                 textName.text = repositoryListInfo.repositoryName
                 textDesc.text = repositoryListInfo.repositoryDesc
                 textLang.text = repositoryListInfo.repositoryLang
            }
```
이렇게 할 수도 있어!!

Reply:
지나가다가 코멘트 남겨용 ^0^..

```suggestion
            binding.apply{
            textName.text = repositoryListInfo.repositoryName
            textDesc.text = repositoryListInfo.repositoryDesc
            textLang.text = repositoryListInfo.repositoryLang
            }
```
이것두 가능합니당 :)

Reply:
두 분 다 좋은 팁 감사합니다!!